base/reducedim.jl

```diff
@@ -200,6 +200,9 @@ function reducedim_init(f::ExtremaMap, op::typeof(_extrema_rf), A::AbstractArray
     return reducedim_initarray(A, region, v0, Tuple{Tmin,Tmax})
 end
 
+# Why is this defined for `max` but not also `min`?
+# It leads to improper behavior, e.g. maximum(abs2, Matrix{Int}(undef, 0, 1), dims=1)
+# Moreover, reducedim_init (above) handles the cases acceptably.
```

Comment:
I am not sure. If you think you can delete this, go for it.

Reply:
The odd thing is the return value.
```
julia> maximum(abs2, Matrix{Int}(undef, 0, 1), dims=1)
1×1 Matrix{Int64}:
0
```
What led me to this discovery was the potential need to match the behavior with `findmax`; currently I do not, as there is no sensible index which could be returned (while also returning a non-empty array).

Whether the lines can be removed, I am not sure. I suspect that a separate PR would be necessary to handle any problems the removal might cause (I'd rather not hold this PR up). Consequently, I am removing the comment form this PR.